src/pages/ChatPage/components/BlockedUserNotice.tsx

```
@@ -0,0 +1,33 @@
+        </h3>
+        <p className="text-sm text-muted-foreground">
+          {t(
+            "blocked.description",
+            "You cannot send messages or see their listings.",
+          )}
+        </p>
+      </div>
+      <button
+        onClick={onUnblock}
+        className="focus:ring-offset-background rounded-xl bg-primary px-10 py-3.5 font-medium text-primary-foreground transition-colors hover:bg-primary/90 focus:outline-none focus:ring-2 focus:ring-primary focus:ring-offset-2"
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)